multi-int-store/build.gradle

```diff
@@ -18,6 +18,8 @@ configurations.all {
 
 dependencies {
     implementation(Dependencies.awsS3())
+    implementation(Dependencies.ingestAPI())
```

Comment:
Why does the mis need the Ingest API?

Reply:
It doesn't. That is a copy and paste error.